### ISSUE TITLE
No module named 'modules.paths'

### ISSUE THREAD

**Issue by bitmanid** [TheLastBen/fast-stable-diffusion#213]
![image](https://user-images.githubusercontent.com/56222671/197386675-3feb3513-7eca-4393-9979-5c6d26474f0a.png)

i had this error when re run the trained model. It worked day before. I had checked the Update Repo but still there is an error

**Comment by TheLastBen** [comment]
Dreambooth colab ?

**Comment by bitmanid** [comment]
@TheLastBen yes

**Comment by TheLastBen** [comment]
The first time you ran it, it worked, then, the second time you get this error ?

**Comment by bitmanid** [comment]
@TheLastBen Yes. i ran it yesterday and it still worked. I have ckpt file for the model. 
I did the steps from 1 to 4 Downloading the model. And then when enter Dreambooth step, i just run "Test the trained Model" using instance name. Are my steps correct?

**Comment by TheLastBen** [comment]
yep I just found the cause, i'll fix it in a minute.

a faster process would be running the first two cells, then directly run the last cell by specifying the INSTANE__NAME, or simply use the AUOTMATIC1111 colab by specifying a path or a link to a trained model.

**Comment by TheLastBen** [comment]
fixed https://github.com/TheLastBen/fast-stable-diffusion/commit/c5430fedfa15a1ec586330802abd7b44c63ec0a5

**Comment by bitmanid** [comment]
@TheLastBen Thank you ben! it works

**Comment by TheLastBen** [comment]
great !
